leasing/fixtures/lease_type.json

```diff
@@ -215,7 +234,8 @@
       "name": "Helsingin Vesi",
       "identifier": "V6",
       "sap_material_code": "20002009",
-      "sap_order_item_number": "",
+      "sap_order_item_number": null,
```

Comment:
This would add inconsistency to sap_order_item_number field in this fixture, because most of the existing values for the field are empty strings instead of null.

Should we align with empty strings or nulls across the board in this fixture?

Reply:
Good point, we should align them. I am thinking if they are empty it is best they would be null. Now if there is a textfield e.g. in admin, I am not sure how storing a value would go with empty string, and how would you set it null/None.
I would prefer these values to be `null` though, and we could perhaps even go as far as removing `blank=True` from these fields.
What do you think @juho-kettunen-nc ?

Reply:
Yes, I agree that representing missing value with `null` would be more "effective" in my opinion. Removing blank = true is also OK for me, we just need to check that modifying and displaying doesn't break.

Reply:
I have changed the values to null, but not making any changes to `blank=True` - for now